1.1/spec/11-Part-08.adoc

```diff
@@ -896,17 +910,17 @@ geof:metricBuffer (geom: ogc:geomLiteral,
                    radius: xsd:double): ogc:geomLiteral
 ```
 
-Returns a geometric object that represents all Points whose distance from `geom` is less than or equal to the `radius` measured in meters. Calculations are in the coordinate reference system of `geom`. This function is similar to <<Function: geof:buffer, `geof:buffer`>>, but does not need a specification of distance unit.
+Returns a geometric object that represents all Points whose distance from `geom` is less than or equal to the `radius` measured in meters. Calculations are in the coordinate reference system of `geom`. This function is similar to <<Function: geof:buffer, `geof:buffer`>>, but does not need a specification of measurement unit.
 
 ==== Function: geof:buffer
 
 ```
 geof:buffer (geom: ogc:geomLiteral, 
-             radius: xsd:double, 
+             radius: rdf:Resource, 
```

Comment:
Why did radius change from xsd:double to rdf:Resource?

Reply:
I don't know! I think this is a mistake by me. I think I've gone too far typing to implement the pairs of metric/non-metric functions and inadvertently set every input for non-metric functions to objects, not doubles, but this function has a `units` parameter so that should be all that's needed with a double numerical value. I'll change this back.